### DISCUSSION TITLE
Status subscribed but no payload

### DISCUSSION THREAD

**Discussion by NewCoder2023** [supabase/supabase#26942] [Questions] [answered]
Hey, 

I use supabase with real-time on and my code gives me SUBSCRIBED as status but I don't get any payload logged if I change something. My realtime is activated though.  Can anyone help me?

    const subscribeToTable = async () => {
        const subscription = supabase
          .channel("All table Names")
          .on(
            "postgres_changes",
            { event: "INSERT", schema: "public", table: "All table Names" },
            (payload) => {
              console.log(payload)
            }  
          )
          .on(
            "postgres_changes",
            { event: "UPDATE", schema: "public", table: "All table Names" },
            (payload) => {
              console.log(payload)
            }
          )
          .on(
            "postgres_changes",
            { event: "DELETE", schema: "public", table: "All table Names" },
            (payload) => {
              console.log(payload)
            }
          )
          .subscribe((status) => {
            console.log("Subscription status:", status);
          });
    
        return () => {
          subscription.unsubscribe();
        };
        
      };
    

Thanks!

**Comment by GaryAustin1** [comment, marked as answer]
This was resolved in Discord.
The issue is realtime does not work for table names with spaces.
![image](https://github.com/supabase/supabase/assets/54564956/924b3dcf-d394-4de0-9d29-8d15025db1ca)
